ibm/mas_devops/roles/sls_install/templates/credential-request.yml.j2

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: masocp-cloud-credential
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: masocp-cloud-credentials
+    namespace: "{{ sls_namespace }}"
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - license-manager:CheckoutLicense
+      - license-manager:GetLicense
+      - license-manager:CheckInLicense
+      - license-manager:ExtendLicenseConsumption
+      - license-manager:ListReceivedLicenses
+      resource: "*"
```

Comment:
missing a newline in the file, atm linting ignored j2 files, but it won't forever, please add newline to end of file